R/elevation.R

```diff
@@ -85,6 +88,10 @@ check_elev_file <- function (elev_file) {
 
 check_bbox <- function (dat, r) {
 
+    if (!requireNamespace ("sp", quietly = TRUE)) {
+        stop ('osm_elevation requires sp package:\n\tinstall.packages("sp")')
```

Comment:
Here too, it's sufficient just to have a simple one-liner, `requireNamespace("sp")`.

Reply:
done